### DISCUSSION TITLE
Alpine is not defined

### DISCUSSION THREAD

**Discussion by rohmatnov** [livewire/livewire#6318] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Import Livewire and Alpine in app.js (Manually bundling Livewire and Alpine)
```js
// resources/js/app.js

import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm'
Livewire.start()
```
2. change show_progress_bar to false in config/livewire.php file
```php
'navigate' => [
    'show_progress_bar' => false,
],
```

Sorry I'm not fluent in English, thank you 

### Additional information

livewire.esm.js:9140 Uncaught ReferenceError: Alpine is not defined at livewire.esm.js:9140:28

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by wizzymore** [comment]
If you want to manually bundle Livewire don't forget to add the following in your layout:
```blade
@livewireScriptConfig()
```
And to also set `'inject_assets' => false,` to false in `config/livewire.php`

**Reply by rohmatnov** [reply]
I've done it but it's still the same

**Reply by lrljoe** [reply]
#7158 <- believe this may relate to your issue!
